elastic/kibana
# [CodeEditor] add support of triple quotes

Closes: #107723

## Summary:

[Vis editor] Triple quote for multiline script support in JSON input 

## Testing notes: 

1. create or open classic visualization e.g. `Area` 
2. set `Average` aggregation for Y-Axis
3. expand `Advanced` panel and set the following `JSON`
```
{
    "script": {
        "lang": "painless",
        "source": """
            return 42
        """
    }
}
```

`Expected result: ` `JSON` should be valid, script should be passed into `ES` request 

## Screens:

![image](https://user-images.githubusercontent.com/20072247/134168128-238a52fb-3f22-4bcb-be04-d810a4c079a9.png)

![image](https://user-images.githubusercontent.com/20072247/134168183-d98cf0e5-c5af-4129-8fdb-867df0e3d9e5.png)
